README.md

```diff
@@ -97,6 +89,7 @@ def testing_concerns(
 ):
 ```
 
+This function will return a `401` error if the team is not authenticated (there is no valid JWT token), and a `403` error if the

```

Comment:
Repeated line?

Reply:
Happened when I pulled and fixed merge conflicts. I did that right before going to work. I'll fix it up.